streams/src/main/java/org/apache/kafka/streams/state/internals/InMemorySessionStore.java

```diff
@@ -168,6 +176,10 @@ public void remove(final Windowed<Bytes> sessionKey) {
                                                                   final long latestSessionStartTime) {
         removeExpiredSegments();
 
+        if (keyFrom == null || keyTo == null) {
```

Comment:
super nit: use `Objects.requireNonNull` for each key with a message along the lines of `keyFrom` is null  / `keyTo` is null